### ISSUE TITLE
Client-side error on setVisible(false) on a parent item

### ISSUE THREAD

**Issue by AlexeiBugai** [vaadin/vaadin-menu-bar-flow#29]
Hi Pekka!

When I do menuItem.setVisible(false), I get next error:
(TypeError) : container is null
Error not always, only for some buttons. Maybe because some items disabled...
But everything works as should.

```
TypeError: container is null[Learn More] contextMenuConnector.js:64:13
    getChildItems http://localhost:8080/frontend/contextMenuConnector.js:64
    items http://localhost:8080/frontend/contextMenuConnector.js:67
    map self-hosted:286
    getChildItems http://localhost:8080/frontend/contextMenuConnector.js:64
    generateItems http://localhost:8080/frontend/contextMenuConnector.js:74
```

P.S. 1.0.0.rc1

**Comment by pekam** [comment]
Hi! I can't reproduce this problem. Can you provide a minimal code example to reproduce it?

**Comment by AlexeiBugai** [comment]
If I give you access to my application via browser, it will help? How I can send login\password to you?

**Comment by pekam** [comment]
No need for that. You can just copy-paste the code where you configure the menu-bar.

**Comment by AlexeiBugai** [comment]
Sorry, my code generates menubar according data in database, and I can't reproduce this in simple way to give you code. I see that code fails when I do setVisible(false) for some buttons with submenu.

In debugger I see that contextMenuConnector fail on line 64:
    const getChildItems = function(parent) {
      const container = getContainer(parent._containerNodeId);
and looks like _containerNodeId is undefined.

**Comment by pekam** [comment]
Unfortunately I can't help when I have no clue of the conditions when the issue occurs.
Maybe you could create the same menu-structure as it generates from the database, but manually with dummy data?

**Comment by AlexeiBugai** [comment]
Ok, I have reproduced problem. It happens from ClickListener...
```
MenuBar menuBarTest = new MenuBar();
  	
MenuItem menuItemTest = menuBarTest.addItem("test");
menuItemTest.setVisible(true);
    	
SubMenu subMenuTest = menuItemTest.getSubMenu();
subMenuTest.addItem("test1");
subMenuTest.addItem("test2");
MenuItem subMenuItemTest = subMenuTest.addItem("test2");
subMenuItemTest.addClickListener(event -> menuItemTest.setVisible(false));

layout.add(menuBarTest);
```

**Comment by pekam** [comment]
Thank you! I was able to reproduce the issue now, although the error looks a bit different for me:
```
Uncaught TypeError: Cannot read property 'children' of null
    at getChildItems (contextMenuConnector.js:77)
```
This actually seems to happen whenever calling `setVisible(false)` on a parent item. So it doesn't exactly relate to those click listeners. I updated the issue header to reflect this.

My mistake was previously that `setVisible` was tested only with an item that had no sub-menu.
